examples/lines_bars_and_markers/hat_graph.py

```diff
@@ -10,42 +10,65 @@
 import matplotlib
 import numpy as np
 import matplotlib.pyplot as plt
+
```

Comment:
Can you fix the comment above while we are here?  Refer ... to: "For discussion of hat graphs, see https://doi.org/10.1186/s41235-019-0182-3"

Reply:
Please feel free to self merge either way

Reply:
I've inlined the link into the first sentence by linking "hat graph" there.